projects/core/src/checkout/facade/checkout-delivery.service.ts

```diff
@@ -34,7 +36,8 @@ export class CheckoutDeliveryService {
   constructor(
     protected checkoutStore: Store<StateWithCheckout | StateWithProcess<void>>,
     protected activeCartService: ActiveCartService,
-    protected userIdService: UserIdService
+    protected userIdService: UserIdService,
+    protected checkoutService: CheckoutService
```

Comment:
We should do this in a non-breaking way.
I suggest to mark the injection with `@Optional`, and as TS optional (`checkoutService?: CheckoutService`).
If the service is provided and present, we can use it.